lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2649,6 +2649,114 @@ def calc_arrow(uvw, angle=15):
 
     quiver3D = quiver
 
+    def voxels(self, filled, color=None):
```

Comment:
The problem with adding `**kwarg` is that it becomes harder to introduce new kwargs to `voxels`, in case they clash with `Poly3DCollection` properties. But I guess `matplotlib` has already gone down that route, so that doesn't seem unreasonable.

If I do that, then I should presumably rename `color` to `facecolor`, to make it clear that `voxels` is in control of that behaviour?